setup.py

```diff
@@ -22,7 +22,7 @@
       license='Public domain',
       classifiers=['Development Status :: 3 - Alpha',
                    'Programming Language :: Python :: 3.6'],
-      # install_requires=['numpy', 'netCDF4', 'xarray'],
+      # install_requires=['numpy', 'netCDF4', 'xarray', 'sqlite3'],
```

Comment:
sqlite3 is part of the default python install and does not need to be in the requires line